tests/test_vms_exist.py

```diff
@@ -5,7 +5,7 @@
 from base import WANTED_VMS
 
 
-EXPECTED_KERNEL_VERSION = "4.14.169-grsec-workstation"
+EXPECTED_KERNEL_VERSION = "4.14.186-grsec-workstation"
```

Comment:
**lgtm**

- [x] Install workstation using this branch
- [x] `make test` tests fail due to version mismatch

I saw `assert kernel_version == EXPECTED_KERNEL_VERSION` fail

~- [ ] Update kernel to  4.14.186 in `securedrop-workstation-buster` using packages from https://github.com/freedomofpress/securedrop-dev-packages-lfs/pull/42~

This step was done for me since by the time I got to this step the lfs pr was merged with the new kernel package

- [x] `make test` tests that check kernel version now pass

I saw `assert kernel_version == EXPECTED_KERNEL_VERSION` no longer fail